app/controllers/application_controller.rb

```diff
@@ -1,2 +1,18 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user, :logged_in?
+
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
```

Comment:
```suggestion
    @current_user ||= User.find_by(id: session[:user_id])
```

Then no need the `if` statement